presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -475,8 +532,16 @@ Session properties
  * **Description:** Use resources that are not guaranteed to be available to a query. This property allows you to exceed the limits of memory available per query and session. It may allow resources to be used more efficiently, but may also cause non-deterministic query drops due to insufficient memory on machine. It can be particularly useful for performing more demanding queries.
 
 
+``small_table_coefficient``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Integer``
+ * **Default value:** ``0.01``
+ * **Description:** See :ref:`small-table-coefficient <tuning-pref-general>`.
+
+
 ``task_concurrency``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^
 
  * **Type:** ``Integer`` (power of 2).
  * **Default value:** ``task.concurrency`` (``1``)
```

Comment:
I believe the default for task.concurrency is now 16 (per Dain's change).
Please update that as well.
CC: @fiedukow